README.md

```diff
@@ -88,13 +88,20 @@ Add the application key provided to you by Placed in the application tag of your
 <meta-data android:name="placed_app_key" android:value="YOUR_APP_KEY" />
 ```
 
-#### Prompting for location permission
+#### Opt-in Dialogue and Prompting for Location Permission
```

Comment:
Should we name this section "Location Permission" to be consistent with iOS? https://github.com/sewichi/ios-placed-sdk#location-permission

Reply:
Not sure, I just followed the document here - https://docs.google.com/document/d/1VDSyPBnRcm-4eFBNYlm8NF3fmSON2ceveDSMptdhrHs/edit